src/main/java/com/takipi/oss/storage/resources/fs/multifetcher/MultiFetcher.java

```diff
@@ -16,31 +16,75 @@
 public class MultiFetcher {
     
     private static final Logger logger = LoggerFactory.getLogger(MultiFetcher.class);
+    private static long maxCacheSize = 33554432;
     
     private final TaskExecutor taskExecutor;
+    private final static S3Cache cache;
+    
+    static {
+        if (maxCacheSize > 0) {
+            cache = new S3CacheImpl(maxCacheSize);
+        }
+        else {
+            cache = new S3Cache() {
+                @Override
+                public String get(String key) {
+                    return null;
+                }
+    
+                @Override
+                public void put(String key, String value) {
+                }
+            };
+        }
+    }
     
     public MultiFetcher(TaskExecutor taskExecutor) {
         this.taskExecutor = taskExecutor;
     }
     
+    public static void setMaxCacheSize(long maxCacheSize) {
+        MultiFetcher.maxCacheSize = maxCacheSize;
```

Comment:
When this function called the code in the `static` section is already run.

Reply:
fixed